.github/pull_request_template.md

```diff
@@ -25,6 +25,19 @@
 
 - ...
 
+## 🔍 Has this modified a publishable library?
```

Comment:
Do we want to call out blocks here? I went looking through our README yesterday for "publish" to find advice and realised we don't talk about blocks in the READMEs, or at least I didn't find it.

Reply:
I've added something to `blocks/README.md`. I don't want to spend much time on this because we should replace the process soon anyway in favour of an automatic one